bin/start-func-test-server.js

```diff
@@ -0,0 +1,191 @@
+/* @flow */
+/* eslint-disable no-console */
+// This starts a docker server for functional tests to access.
+// The script waits for the server to start and prints some logs
+// to help with debugging.
+
+const fs = require('fs');
+const path = require('path');
+const childProcess = require('child_process');
+
+const root = path.join(__dirname, '..');
+console.log(`Working directory: ${root}`);
+const containerIdFile = path.join(root, 'docker-container-id.txt');
+
+// TODO: make these configurable when we need to start
+// servers for multiple apps.
+const appInstance = 'disco';
```

Comment:
Using `process.env.NODE_APP_INSTANCE` would be a pretty easy fix here I think and buy us most of that. I think it's already passed to the tests on travis... if not you could easily add it to the env vars.